b2b-sample/README.md

```diff
@@ -1,6 +1,8 @@
 # WSO2 B2B Organization Samples
 
-Here is a list of sample applications to try out B2B Scenarios.
+## ⚠️ Read this first
+
+1. The root of the sample-is repository  will be referred to `<SAMPLE_IS_HOME>` throughout this document.
```

Comment:
`will be referred to <SAMPLE_IS_HOME>` -> `will be referred to as <SAMPLE_IS_HOME>`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/e4bb0e04657cd72a2db2c08d4631070f07ae9131